build.sbt

```
@@ -1,7 +1,7 @@
 organization in ThisBuild := "org.julienrf"
 
-scalaVersion in ThisBuild := "2.13.0"
-val crossScalaV = Seq( "2.13.0", "2.12.10", "2.11.12")
+scalaVersion in ThisBuild := "2.13.2"
+val crossScalaV = Seq( "2.13.2", "2.12.12", "2.11.12")
```

```suggestion
val crossScalaV = Seq( "2.13.2", "2.12.11")
```

It seems that cats has dropped support for 2.11?

---

yes, let's drop it